procurement/forms.py

```diff
@@ -34,6 +35,17 @@ def clean_pc(self):
                     return gss_codes
         return pc
 
+class HomePagePostcodeAndCouncilForm(PostcodeForm):
+    def clean_pc(self):
+        pc = super().clean_pc()
+        council = Council.objects.filter(name__iexact=pc).first()
```

Comment:
As you mentioned in your comment, my instinct is that this should be `name__icontains=pc`. True, most people will never reach this point in the code, because they’ll already have been redirected by the JavaScript autocomplete. But on the off chance that someone has JavaScript disabled, and submits the form with something like "liverpool" in the search field, I feel like we should do something more useful than just showing them an "invalid council" error message.

This _does_ raise an issue, though, that a substring filter may return multiple matching councils (eg: "liverpool" in my example above, could be Liverpool City Council or Liverpool City Region Combined Authority) and just calling `.first()` might not result in the council the user intended … so we’d probably need to present a disambiguation page, somehow. Sigh.

Maybe leave it as `name__iexact` for now, and open a ticket to replace it with something better, if we go on to develop the alpha further?